prototypes/styletransfer/model.py

```diff
@@ -67,10 +68,10 @@ def feature_representations(
     ) -> Tuple[tf.Tensor, tf.Tensor]:
 
         content_rep, style_rep = self.feature_representations_batch(
```

Comment:
Note to myself: remove `feature_representations_batch`, we don't use batches in style transfer, I assume